### ISSUE TITLE
[A11Y] The contrast between the "Confirm" button text label and the background needs adjustments

### ISSUE THREAD

**Issue by cipriansv** [webcompat/webcompat.com#3422]
**Browser / Version:** Firefox Nightly 81.0a1 (2020-07-27), Chrome 84.0.4147.89
**Operating System:** macOS 10.15.6

**Steps to Reproduce:**

1. Navigate to https://webcompat.com/issues/new

2. Observe the "Confirm" button text label.

**Expected Behavior:**
There should be good contrast between the text label and the button's background.

**Actual Behavior:**
The contrast is not so good.

**Notes:**

1. Screenshot attached.
2. According to "Inspector>Accessibility" tab, the text button does not meet WCAG standard for accessible text.

![image](https://user-images.githubusercontent.com/45160931/88673770-af99a800-d0f1-11ea-8fc5-f045b6223241.png)

3. When the button is enabled the issue is fixed:

![image](https://user-images.githubusercontent.com/45160931/88674871-005dd080-d0f3-11ea-9354-37bac7feefbc.png)

Watchers:
@softvision-oana-arbuzov
@cipriansv

Affected area:
```
<button class="button issue-btn next-url" disabled="">Confirm</button>
```

**Comment by softvision-oana-arbuzov** [comment]
I can confirm this is fixed.
![image](https://user-images.githubusercontent.com/12184325/93484462-9ac7dc80-f90a-11ea-9dbc-a608b7e9cafb.png)

**Tested with:**
Browser / Version: Firefox Nightly 82.0a1 (2020-09-16) 
Operating System: Windows 10 Pro
